### DISCUSSION TITLE
https://angular.love/przywracanie-pozycji-przewijania-scroll-w-angularze

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#290] [General]
# https://angular.love/przywracanie-pozycji-przewijania-scroll-w-angularze

Angular.love - ciekawostki oraz rozwiązania dla średnio-zaawansowanych oraz zaawansowanych developerów Angulara.

https://angular.love/przywracanie-pozycji-przewijania-scroll-w-angularze

**Comment by kkachniarz220** [comment]
Świetny artykuł! Warto wspomnieć, że wykorzystanie resolverów automatycznie rozwiązuje ten problem :)
